index.html

```diff
@@ -0,0 +1,145 @@
+                    </li>
+                    <li <a href="" class="log-in padding">LOGIN</a></li>
+                </ul>
+            </div>
+        </nav>
+    </div>
+
+    <div>
+        <div class="hero-section">
+
+            <img class="img-hero m-t" src="../assests/sale.jpg" alt="book reading">
+            <img class="img-hero m-t" src="../assests/heroo.jpg" alt="book reading">
+
+        </div>
+
+    </div>
+<div class="category">
+<p>SHOP BY CATEGORY</p>
+</div>
+    <section class="card-categories m-t5">
+
+        <a href="" class="cards">
+            <div>
+                <img src="/assests/women.jpg" class="card-image" alt="John">
+            </div>
+            <h2 class="card-heading">Women</h2>
+        </a>
+
+        <a href="" class="cards">
+            <div>
+                <img src="/assests/men.jpg" class="card-image" alt="John">
+            </div>
+            <h2 class="card-heading">Men</h2>
+        </a>
+
+        <a href="" class="cards">
+            <div>
+                <img src="/assests/accesories.jpg" class="card-image" alt="John">
+            </div>
+            <h2 class="card-heading">Accessories</h2>
+        </a>
+
+        <a href="" class="cards">
+            <div>
+                <img src="/assests/shoes.jpg" class="card-image" alt="John">
+            </div>
+            <h2 class="card-heading">Shoes For Men</h2>
+        </a>
+
+
+
+
+    </section>
+
+
+    <div class="footer-container">
+        <footer>
+            <div class="container-footer padding-footer">
+
+                <div class="item">
+                    <h2>Main Menu</h2>
+                    <ul>
+                        <li><a href="#">HOME</a></li>
+                        <li><a href="#">ABOUT US</a></li>
+                        <li><a href="#">BLOG</a></li>
+                    </ul>
+                </div>
+                <div class=" item">
+                    <h2>Useful Links</h2>
+                    <ul>
+                        <li><a href="#">FAQ</a></li>
+                        <li><a href="#">PRIVACY POLICY</a></li>
+                        <li><a href="#">TERMS AND CONDITIONS</a></li>
+                        <li><a href="#">RETURN AND REFUND POLICY</a></li>
+                    </ul>
+                </div>
+                <div class=" item">
+                    <h2>Contact Us</h2>
+                    <ul>
+                        <li><a href="#">CUSTOMER SUPPORT</a></li>
+                    </ul>
+                </div>
+                <div class=" item">
+                    <h2>Social Media</h2>
+                    <ul>
+                        <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a> </li>
+                        <li><a href="#"><i class="fab fa-twitter"></i></a> </li>
+
+
+                    </ul>
+                </div>
+
+
+            </div>
+        </footer>
+    </div>
+</body>
+
+</html>
```

Comment:
add a new line at the end of file.

Reply:
Add a new line at the end of every file. It is a good practice. Also, use any code formatter such as prettify or beautify. Other than that, UI looks amazing. Great work done.